### PR TITLE
Spruce up README examples

### PR DESCRIPTION
**Pull request by lucperkins**
The examples currently use incorrect syntax for Nix comments.

### CODE REVIEW

**Comment by cole-h** [comment]
Not quite sure what you mean, `/* */` are completely valid Nix comments:

```nix
nix-repl> hi = /* hello there */ "world"

nix-repl> hi
"world"
```

and https://github.com/NixOS/nixpkgs/blob/cfdae65b07b95ddd0bddd3af65e171fffd2f85e2/lib/asserts.nix#L5-L37

**Comment by lucperkins** [comment]
@cole-h That's news to me! I don't have a strong preference for either style of comment. But the first example is indeed wrong. As for `@ inputs`, I can be persuaded to keep it but I feel like that style is slightly more likely to cause confusion in comparison to `{ ... }`.

**Review by cole-h — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ Install and manage Determinate Nix.
 ```nix
 {
   inputs.nix.url = "https://flakehub.com/f/DeterminateSystems/nix/2.0";
+  inputs.nixpkgs.url = "https://flakehub.com/f/DeterminateSystems/nixpkgs/0.2405.*";
```
Comment (cole-h) [inline README.md:10]: We don't publish nixpkgs to `DeterminateSystems/nixpkgs`

File: README.md [hunk 2]
```diff
@@ -7,15 +7,16 @@ Install and manage Determinate Nix.
...
 
-  outputs = { ... }: @ inputs {
-    nixosConfigurations.default = nix-darwin.lib.darwinSystem {
+  outputs = { nix, nixpkgs, ... }: {
+    nixosConfigurations.default = nixpkgs.lib.nixosSystem {
```
Comment (cole-h) [inline README.md:13]: `nixosSystem` requires a `system =` argument specified.

File: README.md (hunk 1 above)
Comment (lucperkins) [reply README.md:10]: Derp

**Review by cole-h — APPROVED** [review]

**Pull request merged**